src/components/CarouselProductCard/textAlignStyles.mobile.tsx

```diff
@@ -0,0 +1 @@
+export const textAlignStyles = 'text-left';
```

Comment:
```suggestion
export const textAlignStyle = 'text-left';
```

назови пожалуйста и файл также, без `s` в конце. Так будет согласно по недавнему конвеншену